ouroboros-consensus-shelley/src/Ouroboros/Consensus/Shelley/Ledger/Query.hs

```diff
@@ -141,49 +140,49 @@ data instance BlockQuery (ShelleyBlock era) :: Type -> Type where
   -- decode it, the client can fall back to pretty printing the actual CBOR,
   -- which is better than no output at all.
   GetCBOR
-    :: BlockQuery (ShelleyBlock era) result
-    -> BlockQuery (ShelleyBlock era) (Serialised result)
+    :: BlockQuery (ShelleyBlock proto era) result
+    -> BlockQuery (ShelleyBlock proto era) (Serialised result)
 
   GetFilteredDelegationsAndRewardAccounts
     :: Set (SL.Credential 'SL.Staking (EraCrypto era))
-    -> BlockQuery (ShelleyBlock era)
+    -> BlockQuery (ShelleyBlock proto era)
              (Delegations (EraCrypto era), SL.RewardAccounts (EraCrypto era))
 
   GetGenesisConfig
-    :: BlockQuery (ShelleyBlock era) (CompactGenesis era)
+    :: BlockQuery (ShelleyBlock proto era) (CompactGenesis era)
 
   -- | Only for debugging purposes, we make no effort to ensure binary
   -- compatibility (cf the comment on 'GetCBOR'). Moreover, it is huge.
   DebugNewEpochState
-    :: BlockQuery (ShelleyBlock era) (SL.NewEpochState era)
+    :: BlockQuery (ShelleyBlock proto era) (SL.NewEpochState era)
 
   -- | Only for debugging purposes, we make no effort to ensure binary
   -- compatibility (cf the comment on 'GetCBOR').
   DebugChainDepState
-    :: BlockQuery (ShelleyBlock era) (SL.ChainDepState (EraCrypto era))
+    :: BlockQuery (ShelleyBlock proto era) (ChainDepState proto)
```

Comment:
I think that yes, the changes in this PR will need to be heavily documented there (albeit this query is possible the least relevant change, since the Debug thing is not guaranteed stable.